core/types/rollup_cost.go

```diff
@@ -387,6 +387,12 @@ func EstimatedL1SizeScaled(costData RollupCostData) *big.Int {
 	return estimatedSize
 }
 
+// EstimatedDASize is the unscalled version of EstimatedDASizeScaled.
```

Comment:
`unscalled` -> `unscaled`

Reply:
now that this is the only public method made this a full comment.